sdks/rust/copy-protos.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
What does this gen used for? I couldn't find this referenced anywhere? Might also be okay to remove with the above comments.

Reply:
Sorry, this script was for my own sanity as going through the `make` scripts forces things like Go being installed etc which I wanted to avoid at all costs, I'll just remove it.